Modules/Devices/src/Devices.Infrastructure.Database.Postgres/Migrations/20240701074627_Init.cs

```diff
@@ -562,6 +562,8 @@ protected override void Up(MigrationBuilder migrationBuilder)
                 table: "Tiers",
                 column: "Name",
                 unique: true);
+
+            migrationBuilder.AddCheckConstraintForAtMostOneActiveDeletionProcessPerIdentity();
```

Comment:
Whoah, I just noticed that you added code to an existing migration. You should NEVER do that. Imagine the migration is already applied to a database. In that case, the change will never be applied to that database.

Reply:
Yeah, You're right. I don't know why I thought this would a good idea.